### ISSUE TITLE
Unexpected Error (on a Vimeo site) Error: Unable to extract info section 

### ISSUE THREAD

**Issue by dpcunningham** [ytdl-org/youtube-dl#31387]
Thanks for the project.  Have been using it for quite some time.  Love it.  

Ran into an unexpected error on a Vimeo link -- thought I'd pass it along.

## Checklist

- [x] I'm reporting a broken site support
- [x] I've verified that I'm running youtube-dl version **2021.12.17**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [x] I've searched the bugtracker for similar issues including closed ones

## Verbose log

```
pc:~/Downloads$ sudo pip install -U youtube-dl
[sudo] password for dpc:                  
Requirement already up-to-date: youtube-dl in /usr/local/lib/python3.8/dist-packages (2021.12.17)
dpc:~/Downloads$ youtube-dl --version
2021.12.17
dpc:~/Downloads$ youtube-dl --verbose https://player.vimeo.com/video/770284902
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['--verbose', 'https://player.vimeo.com/video/770284902']
[debug] Encodings: locale UTF-8, fs utf-8, out utf-8, pref UTF-8
[debug] youtube-dl version 2021.12.17
[debug] Python version 3.8.10 (CPython) - Linux-5.4.0-126-generic-x86_64-with-glibc2.29
[debug] exe versions: ffmpeg 4.2.7, ffprobe 4.2.7
[debug] Proxy map: {}
[vimeo] 770284902: Downloading webpage
ERROR: Unable to extract info section; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/youtube_dl/YoutubeDL.py", line 815, in wrapper
    return func(self, *args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/youtube_dl/YoutubeDL.py", line 836, in __extract_info
    ie_result = ie.extract(url)
  File "/usr/local/lib/python3.8/dist-packages/youtube_dl/extractor/common.py", line 534, in extract
    ie_result = self._real_extract(url)
  File "/usr/local/lib/python3.8/dist-packages/youtube_dl/extractor/vimeo.py", line 651, in _real_extract
    config = self._parse_json(self._search_regex(
  File "/usr/local/lib/python3.8/dist-packages/youtube_dl/extractor/common.py", line 1012, in _search_regex
    raise RegexNotFoundError('Unable to extract %s' % _name)
youtube_dl.utils.RegexNotFoundError: Unable to extract info section; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.

```

## Description

Just reporting an unexpected `Unable to extract info section;` on a vimeo site, b/c the app asked me to do so.

WRITE DESCRIPTION HERE

Standard usage, unexpected error -- submitting a log with version verification and detailed error messages.

Thanks for supporting an awesome project!

**Comment by october262** [comment]
remove the words player & video so it looks like this "https://vimeo.com/770284902"
the video will now download

**Comment by dpcunningham** [comment]
Thanks, @dirkf.  I actually did perform a (admittedly fast) check, but I didn't spot the work-around.  

I can also see it must be time-consuming, hence frustrating for you (as evidenced by some of the "really"..." remarks in other issues.)  So I just wanted you to know that we SUPER appreciate the support work you do!  

But it also seems like a legitimate re-parsing issue that might want to be addressed.  I imaging the requests will keep coming in, just because it's not easy to search for similar errors (more on that below).  

Well, it's actually NOT that hard, but it's just not that well supported by the GH search functionality.  And your typical end-users are likely not going to do well using the search tools as-is.  Not without more help (and more on that below).

Also, in my defense, as a "busy-end-user", the original work-around (per issue #31311) is buried deep within a workflow conversation that I NEVER would have dove into, especially given the issue title is about a RegEx problem (which to me would not look relevant at all).  

Now, to be fair, I didn't realize I could drop the error string into the GH issue search panel directly, because... well... it's a crappy interface.  And that's NOT your fault at all, it's a GH "thang".

If I could make a suggestion, maybe an enhanced section of "resolving bugs" might be useful?  One where y'all say: "Hey, silly, before you waste our time, cut the error string, go over to this page, paste it into the search text edit box (and ignore the fact that it looks like a selector-item, not a search string text edit box).  

Maybe that's already there, but I'd argue that if it is, it's not obvious enough.  I certainly didn't see it., and in my first pass through, 

I also missed the fact that the search page had a useful text-edit field (again, on first impression, it looks like a choice selector, not a text-edit field.  And I am an experienced GitLab user.  

In other words, a usability issue completely NOT the fault of the `youtube-dl` project at all.   

Let me know if I can help here.  I'd be willing to have a go at re-working project documentation to perhaps yield better end-user behaviors.

And, no matter what, I wanted to give you a big, Big, BIG -- THANK YOU -- for the support work you do!

**Comment by dpcunningham** [comment]
Thanks, @dirkf.  I actually did perform a (admittedly fast) check, but I
didn't spot the work-around.

I can also see it must be time-consuming, hence frustrating for you (as
evidenced by some of the "really"..." remarks in other issues.)  So I just
wanted you to know that we SUPER appreciate the support work you do!

But it also seems like a legitimate re-parsing issue that might want to be
addressed.  I imaging the requests will keep coming in, just because it's
not easy to search for similar errors (more on that below).

Well, it's actually NOT that hard, but it's just not that well supported by
the GH search functionality.  And your typical end-users are likely not
going to do well using the search tools as-is.  Not without more help (and
more on that below).

Also, in my defense, as a "busy-end-user", the original work-around (per
issue #31311) is buried deep within a workflow conversation that I NEVER
would have dove into, especially given the issue title is about a RegEx
problem (which to me would not look relevant at all).

Now, to be fair, I didn't realize I could drop the error string into the GH
issue search panel directly, because... well... it's a crappy interface.
And that's NOT your fault at all, it's a GH "thang".

If I could make a suggestion, maybe an enhanced section of "resolving bugs"
might be useful?  One where y'all say: "Hey, silly, before you waste our
time, cut the error string, go over to this page, paste it into the search
text edit box (and ignore the fact that it looks like a selector-item, not
a search string text edit box).

Maybe that's already there, but I'd argue that if it is, it's not obvious
enough.  I certainly didn't see it., and in my first pass through,

I also missed the fact that the search page had a useful text-edit field
(again, on first impression, it looks like a choice selector, not a
text-edit field.  And I am an experienced GitLab user.

In other words, a usability issue completely NOT the fault of the
`youtube-dl` project at all.

Let me know if I can help here.  I'd be willing to have a go at re-working
project documentation to perhaps yield better end-user behaviors.

And, no matter what, I wanted to give you a big, Big, BIG -- THANK YOU --
for the support work you do!
